intelmq/bin/intelmqctl.py

```diff
@@ -855,6 +855,7 @@ def __init__(self, interactive: bool = False, return_type: str = "python", quiet
             parser_list.add_argument('--non-zero', '--quiet', '-q', action='store_true',
                                      help='Only list non-empty queues '
                                           'or the IDs of enabled bots.')
+            parser_list.add_argument('--count', '--sum', action='store_true', help='Show total events of all queues')
```

Comment:
Please add a trailing dot and rephrase it a bit. Maybe "Only show the total number of messages in queues. Only valid for listing queues."